includes/gateways/stripe/class-give-stripe.php

```diff
@@ -59,6 +59,21 @@ public function __construct() {
 		 */
 		public function includes() {
 
+			// Include admin files.
+			require_once GIVE_PLUGIN_DIR . 'includes/gateways/stripe/includes/admin/admin-helpers.php';
+			require_once GIVE_PLUGIN_DIR . 'includes/gateways/stripe/includes/admin/admin-actions.php';
```

Comment:
@mehul0810 Tried to right show function as much as possible. As I can see. I can break `includes` function into `frontend_files` and `admin_files`.


We can also refactor this specific condition to `load_stripe_sdk`.
```
	if ( 'composer' === $stripe_sdk_compatibility ) {
				require_once GIVE_PLUGIN_DIR . 'vendor/autoload.php';
			} elseif ( 'manual' === $stripe_sdk_compatibility ) {
				require_once GIVE_PLUGIN_DIR . 'vendor/stripe/stripe-php/init.php';
			}
```

I am suggesting to refactor big functions because it reduces the complexity of function and also makes them readable.

Reply:
Refactored `includes` function with multiple smaller functions